.gitignore

```diff
@@ -3,6 +3,7 @@
 
 # Ignore configuration files that may contain sensitive information.
 settings.php
+settings.*.php
```

Comment:
We could simplify this rule into simply `settings*.php`. Do you think that would do any harm?